phenotype.gemspec

```diff
@@ -2,19 +2,19 @@ require './lib/phenotype/version'
 
 Gem::Specification.new do |s|
   s.name = 'phenotype'
-  s.homepage = ''
+  s.homepage = 'https://github.com/rentpath/phenotype'
   s.version = Phenotype::VERSION
   s.authors = ['Rentpath']
   s.summary = "phenotype-#{Phenotype::VERSION}"
   s.description = 'versioning gem that works with rails and rack'
-  s.license = 'NA'
+  s.license = 'MIT'
   s.files = Dir['lib/**/*']
   s.rdoc_options = ['--charset=UTF-8']
   s.require_paths = ['lib']
 
-  s.add_development_dependency 'pry', '~> 0.10'
-  s.add_dependency 'rack', '~> 1'
-  s.add_dependency 'require_all', '~> 1.3'
-  s.add_development_dependency 'rake', '~> 10.5'
-  s.add_development_dependency 'rspec', '~> 3.4'
+  s.add_dependency 'rack', '>= 1'
+  s.add_dependency 'require_all', '>= 1'
+  s.add_development_dependency 'pry', '>= 0.10'
+  s.add_development_dependency 'rake', '>= 10'
+  s.add_development_dependency 'rspec', '>= 3'
```

Comment:
was the intention to allow older versions than what was previously supported for `require_all`, `rake` and `rspec`?
`~> 3.4` is equivalent to `3.4 >= x < 3.5` but you are now allowing `rspec` older than `3.4`

Reply:
good point, will update

Reply:
`~> 3.4` is equivalent to `3.4 <= x < 4.0`, but Eric is right that `>= 3` will allow older versions of `rspec`.

Reply:
> ~> 3.4 is equivalent to 3.4 <= x < 4.0
d'oh ur right